README.md

```diff
@@ -182,13 +182,26 @@ limitd.takeElevated(type, key, { count, configOverride, erlIsActive }, (err, res
 -  `count`: the amount of tokens you need. This is optional and the default is 1.
 -  `configOverride`: caller-provided bucket configuration for this operation
 -  `erlIsActiveKey`: (string) the identifier of the ERL activation for the bucket.
+-  `erlQuota`: (object)
```

Comment:
Do you think it makes sense to merge the ERL configuration into a single ERL object?

Reply:
I'm ok with breaking the object into its attributes. The only reason I considered the object was because it sounded like the bucket configuration or the elevated_limits configuration which both are objects.
With an object it would be 
`erlQuota: { key: 'someQuotaKey', per_calendar_month: 192 }`

Without an object, I forsee something like:
` {..., erlQuotaKey: 'someQuotaKey', erlQuotaPerCalendarMonth:192 }`

Reply:
fixed